document/chapters/04-simd_bvh.tex

```diff
@@ -290,12 +290,12 @@ \subsection{Transport Tests}
 was determined via debugging values that this discrepancy is caused by a single
 particle ending in EmDAG-MCNP near a surface of cell 2 while in DAG-MCNP the
 particle crosses into cell 3 before abruptly terminating though it still
-contributing slightly to the tally in cell 3. It is believed that this
-difference in tally result is the result of a systematic difference between
-Embree and MOAB's ray fire conventionality rather than being result of the
-double to single floating point conversion of the model that occurs when using
-EmDAG - though this conversion in precision is shown to be problematic in other
-ways in Section \ref{sec:emdag_limitations}.
+contributing slightly to the tally in cell 3. This difference is caused by a
+difference in ray fire convention between MOAB and Embree rather than being
+result of the double to single floating point conversion of the model that
+occurs when using EmDAG. Ray-triangle intersection results for rays with origins
+numerically on a triangle are considered intersecting in MOAB but not in Embree.
```

Comment:
This is definitely much more clear, but as someone familiar with the details, I'm still not sure I understand.  Don't most rays have origins on triangles as they cross surfaces?  do you mean a triangle edge? or a triangle vertex?

Reply:
Had to dig into old tests to remember... you're right, it's node hits. When I implemented EmDAG they were relying on the Moeller triangle test rather than the Pluecker test. Node hits in DGAMC cause zero distance intersections where in EmDAG small distances traveled will be reported instead, making contributions to the flux values different.

I think for the purposes of the discussion here, I can simply say that differences in the triangle intersection tests being used caused changes in the flux contributions. 

Sidenote: Embree has since added the Pluecker test for their "robust mode".

Reply:
I'm updating this wording, but I'm wondering if a couple of quick sections in the background on the differences between Moeller and Pluecker tests would be a good idea here.

**Not** trying to drag this out, but I want to be thorough.

Reply:
I don't think you need to worry about this.  The current wording provides an appropriate amount of description and deeper understanding of triangle tests is not necessarily important within the scope of your work.

Reply:
Ok sounds good.